data.sql

```diff
@@ -91,4 +91,11 @@ VALUES
 ((SELECT id FROM animals WHERE name='Boarmon'), (SELECT id FROM vets WHERE name='Maisy Smith'),'2020-02-27'),
 ((SELECT id FROM animals WHERE name='Boarmon'), (SELECT id FROM vets WHERE name='Maisy Smith'),'2020-08-03'),
 ((SELECT id FROM animals WHERE name='Blossom'), (SELECT id FROM vets WHERE name='Stephanie Mendez'),'2020-05-24'),
-((SELECT id FROM animals WHERE name='Blossom'), (SELECT id FROM vets WHERE name='William Tatcher'),'2021-01-11');
\ No newline at end of file
+((SELECT id FROM animals WHERE name='Blossom'), (SELECT id FROM vets WHERE name='William Tatcher'),'2021-01-11');
+
+-- Adding data to increase the execution
+
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
- [ ] Great work creating this data. After populating the necessary tables with these data, you ought to write the following queries in the _**queries.sql**_ file:

- ` EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;`
- `EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;`
- `EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';`

The outcome of each of the following queries should have an execution time that is **_greater or equal_** to `1000ms`. 👍 

Then you take a screenshot and attach it to your pull request. After these, you then run the SQL statements that apply indexes to the special columns and then run each of the queries above again. 😎 

Thanks to the indexes applied, their execution time should be **_significantly lower_** than `1000ms`. Take the screenshots and attach them to your pull request summary. Kindly implement these changes. 👍 😎 

#### Hint:

- Run the queries before applying the indexes. Take screenshots of **_execution time above 1000ms_**
- Apply the indexes
- Run the queries again. Take screenshots of execution time **_significantly below 1000ms_**.